docs/user-guide/simulator_integration.md

```diff
@@ -2,20 +2,20 @@
 
 At some point, maybe from the very beginning of your time with HELICS co-simulation, you'll have an interest or need to include a simulator in your co-simulation that HELICS doesn't support. Maybe it's an existing open-source simulator, maybe it's commercial software, maybe it's a small controller simulator you'd like to test in an existing model. HELICS has been designed to make it as easy as possible to integrate a new simulator. Before writing code, though, it is important to more specifically define the task.
 
-## Simulator Integration Clarifying Questions##
+## Simulator Integration Clarifying Questions ##
   1. **What is the nature of the code-base being integrated?** Is this open-source code that can be fully modified? Is it a simulator, perhaps commercial, that provides an API that will be used?  How much control do you, the integrator, have in modifying the behavior of the simulator?
   2. **What programming language will be used?** - HELICS has bindings for a number of languages and the one that is best to use may or may not be obvious. If you're integration of the simulator will be through the API of the existing simulator, then you'll likely be writing a standalone executable that wraps that API. You may be constrained on the choice of languages based on the method of interaction with that API. If the API is accessed through a network socket then you likely have a lot of freedom in language choice. If the API is a library that you call from within wrapper, you will likely be best of using the language of that library.
 
-  If you're writing your own simulator then you have a lot more freedom and the language you use may come down to personal preference and/or performance requirements of the federate.
+      If you're writing your own simulator then you have a lot more freedom and the language you use may come down to personal preference and/or performance requirements of the federate.
 
-  The languages currently supported by HELICS are:
-   - C++
-   - C
-   - Python (2 and 3)
-   - Java
-   - MATLAB
-   - Octave
-   - C# (somewhat limited as of yet)
+      The languages currently supported by HELICS are:
+      - C++
+      - C
+      - Python (2 and 3)
+      - Java
+      - MATLAB
+      - Octave
+      - C# (somewhat limited as of yet)
```

Comment:
Add some of the language Dheepak was working with Julia or nim(@kdheepak) any more?

Reply:
Added Julia and Nim to the list.